docker-compose.yml

```diff
@@ -1,52 +1,76 @@
-version: '2.1'
+version: '3'
+  
 services:
-
-  blacklight:
-    image: yalelibraryit/dc-blacklight:${BLACKLIGHT_TAG}
-    command: /bin/sh -c "cd /home/app/webapp && sleep 10 && rake yale:load_voyager_sample_data && /sbin/my_init"
-    env_file:
-      - .env
+  iiif_manifest:
+    image: yalelibraryit/dc-iiif-manifest:${IIIF_MANIFEST_VERSION}
     ports:
-      - '3000:3000'
-    depends_on:
-      - solr
-      - db
-      - iiif_image
-      - iiif_manifest
-
-  iiif_image:
-    image: yalelibraryit/dc-iiif-cantaloupe:${CANTALOUPE_TAG}
-    env_file:
-      - .env
+      - '80:80'
+    logging:
+      driver: awslogs
+      options:
```

Comment:
I'm trying to figure out how to run this locally, and the logging is the current blocker. Even when I set the environment variables in the readme, I still get 
```bash
ERROR: for yul-dc-camerata_solr_1  Cannot create container for service solr: unknown log opt 'awslogs-stream-prefix' for awslogs log driver
```

Reply:
There is no way to run it locally.  We need to get logging fixed - there's still a lot of refinement to be done here - think of this as shameless green.  But before adding too many more refinements, I want to make sure that this all works somewhere besides my development environment - it already feels like there's a lot of scaffolding here that the rest of the team hasn't been exposed to.